bin/limitd

```diff
@@ -48,6 +48,8 @@ try{
 }
 
 const logger = getLogger(config.log_level);
+const eventLogger = getEventLogger(logger);
+eventLogger.watch(process);
```

Comment:
👍

Reply:
Don't you need to handle error event on server too? or you prefer to leave the uncaught to be caught by the eventLogger